JM/CF/Scripts/1_Core/CommunityFramework/Logging/CF_Log.c

```diff
@@ -55,6 +55,16 @@ class CF_Log
 		if (Level > CF_LogLevel.ERROR) return;
 
 		PrintFormat("[ERROR] %1", string.Format(message, param1, param2, param3, param4, param5, param6, param7, param8, param9));
+
+		string dump = "";
+		DumpStackString(dump);
+		array<string> outputs = new array<string>();
+		dump.Split("\n", outputs);
+
+		for (int i = 1; i < outputs.Count(); i++)
+		{
+			Print("\t" + outputs[i]);
+		}
```

Comment:
Hide behind a param if the stack shall be outputted as well. I think default should be false and only include the PrintFormat

Reply:
Can't add behind a param, should I add two new methods called `TraceError` and `TraceCritical`?

I would prefer Error and Critical just always output to the stack log though. This is because as the docs say these two log methods are only used when execution is halted. So the user and modder should always know the cause.

Reply:
Ok fair enough, the prototype does not allow for that to be easily added.
Leave it as it is then.